src/main/java/com/dyescape/dyescapebot/entity/discord/WarningAction.java

```diff
@@ -0,0 +1,102 @@
+package com.dyescape.dyescapebot.entity.discord;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@IdClass(WarningAction.WarningActionKey.class)
+@Table(name = "warning_actions")
+public class WarningAction implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    // -------------------------------------------- //
+    // COLUMNS
+    // -------------------------------------------- //
+
+    @Id
+    @Column(name = "server_id")
+    private long server;
+
+    @Id
+    @Column(name = "points")
```

Comment:
Multiple primary keys. See above

Reply:
In this case, the server ID and the warning actions are primary keys. This makes sure that a server can only have one action on a given warning point. Again here, the combination of the server ID and amount of points is the primary key. This is done so the database will make sure all data stays fully correct.

Reply:
makes sense